modules/pam_limits/limits.conf.5.xml

```diff
@@ -227,6 +227,13 @@
                   user with <emphasis>uid=0</emphasis>)</para>
               </listitem>
             </varlistentry>
+            <varlistentry>
+              <term><option>nonewprivs</option></term>
+              <listitem>
+                <para>disable acquiring new privileges by invoking
```

Comment:
please add `value of 0 or 1; if set to 1` ....

Reply:
Done.

For some reason I'm not receiving email notifications from github for these comments, otherwise I would have updated this quicker.